starter/js/atm.js

```diff
@@ -1,12 +1,45 @@
 //Begin with the document ready function
+$(document).ready(function() {
 
-    //Checking account deposit function
 
+    //Checking account deposit function
       //On click of the depositChecking button
+    $('#depositChecking').click(function(event) {
+      // console.log(event);
+            //Get value from the amountChecking input field
+        var depositAmount = $('#amountChecking').val();
+        // console.log(depositAmount)
+        //Take that value and add it to the existing value in the checkingBalance div
+        var checkingBal = $('#checkingBalance').text();
+        // console.log(checkingBal);
+
+        // console.log(strValue);
+        var totalBalance =  parseInt(depositAmount) + parseInt(checkingBal.replace('$',''));
+        $('#checkingBalance').text('$' + totalBalance);
+        // console.log(totalBalance);
+
+        });
+          $('#withdrawChecking').click(function(event) {
+      // console.log(event);
+            //Get value from the amountChecking input field
+        var withdrawAmount = $('#amountChecking').val();
+        // console.log(depositAmount)
+        //Take that value and add it to the existing value in the checkingBalance div
+        var checkingBal = $('#checkingBalance').text();
+        // console.log(checkingBal);
 
-        //Get value from the amountChecking input field
+        // console.log(strValue);
+        var totalBalance = parseInt(checkingBal.replace('$','')) - parseInt(withdrawAmount);
+        // $('#checkingBalance').text('$' + totalBalance);
 
-        //Take that value and add it to the existing value in the checkingBalance div
+        if (totalBalance <= withdrawAmount) {
```

Comment:
If we start the account at $100 and withdraw $10, the totalBalance value is $90 and withdrawAmount is $10. Therefore, this condition would be false because the totalBalance is larger than withdrawAmount. You want to reverse the condition or use a different comparison here.